### PR TITLE
[WIP] Fix #33

### LINKED ISSUE #33: `Property not found in...` errors not reported
If I have a function

``` js
function foo(bar: {baz: string}) {...}
```

And I call it with

``` js
foo(1234);
```

An error is reported as expected (due to the mismatched type). But if I call it with

``` js
foo({xyz: 'abc'});
```

An error is reported by Flow but not by this plug-in. Seems that the code filtering out issues is incorrect: https://github.com/amilajack/eslint-plugin-flowtype-errors/blob/master/src/collect.js#L76 as `filepath` and `message[0].path` don't match in this case.

### PR DESCRIPTION
**Pull request by jdmota**
Trying to see if I can properly fix #33 this time...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build": "cross-env NODE_ENV=production babel src --out-dir dist",
     "lint": "eslint --format=node_modules/eslint-formatter-pretty src test *.js",
     "preversion": "npm run build",
-    "spec": "npm run build && jest test/*.spec.js",
+    "spec": "npm run build && jest test/format.spec.js",
```
Comment (amilajack) [inline package.json]: Let's keep this to `*.spec.js`. There's probably going to be more tests added here in the future

File: src/collect.js [hunk 2]
```diff
@@ -114,14 +114,33 @@ function executeFlow(stdin, root, filepath) {
...
+        message[2] = tmp;
+        const tmp2 = message[0].descr;
+        message[0].descr = message[2].descr;
+        message[2].descr = tmp2;
+      }
+      return message;
+    })
     // Temporarily hide the 'inconsistent use of library definitions' issue
-    .filter(({ message }) => (
+    .filter((message) => (
```
Comment (amilajack) [inline src/collect.js]: For consistency, lets remove the parens here

File: src/collect.js [hunk 3]
```diff
@@ -114,14 +114,33 @@ function executeFlow(stdin, root, filepath) {
...
       !message[0].descr.includes('inconsistent use of') &&
       pathModule.resolve(root, message[0].path) === fullFilepath &&
       message[0].descr &&
       message[0].descr !== ''
     ))
-    .map(({ message }) => {
+    .map((message) => {
```
Comment (amilajack) [inline src/collect.js]: For consistency, lets remove the parens here

**Comment by amilajack** [comment]
Merged! Thanks for this! 🎉🎉🎉

**Comment by jdmota** [comment]
I'm not sure if this kind of issue could happen in other situations... But at least this particular instance is fixed.

Btw, don't know if you noticed, but I changed the test script to `jest test/format.spec.js` (instead of `jest test/*.spec.js` because, on Windows, jest was not finding the test file.

**Pull request merged**